### ISSUE TITLE
Problem with spamassassin packages

### ISSUE THREAD

**Issue by stefaweb** [mailwatch/mailwatch-install-script#37]
Still trying the script.

Still with fresh Debian Stretch fresh install + mariadb 10.1 installed + postfix installed (not configured).

After ran setup.sh, I found all these things in /usr/local/bin:

```
-r-xr-xr-x 1 root staff   3871 juil. 14 23:55 fix_latin
-r-xr-xr-x 1 root staff   2027 juil. 14 23:39 ip2cc
-rwxr-xr-x 1 root staff   3538 juil. 15 00:05 mailwatch_db_clean.php
-rwxr-xr-x 1 root staff   8690 juil. 15 00:05 mailwatch_geoip_update.php
-rwxr-xr-x 1 root staff   7484 juil. 15 00:05 mailwatch_quarantine_maint.php
-rwxr-xr-x 1 root staff   2524 juil. 15 00:05 mailwatch_quarantine_report.php
-r-xr-xr-x 1 root staff   4953 juil. 14 23:41 sa-awl
-r-xr-xr-x 1 root staff  14715 juil. 14 23:40 sa-check_spamd
-r-xr-xr-x 1 root staff  22065 juil. 14 23:40 sa-compile
-r-xr-xr-x 1 root staff  45592 juil. 14 23:41 sa-learn
-r-xr-xr-x 1 root staff  66799 juil. 14 23:54 sa-update
-r-xr-xr-x 1 root staff  30090 juil. 14 23:41 spamassassin
-r-xr-xr-x 1 root staff 143288 juil. 14 23:41 spamc
-r-xr-xr-x 1 root staff 129256 juil. 14 23:40 spamd
-r-xr-xr-x 1 root staff   9397 juil. 14 23:39 spfd
-r-xr-xr-x 1 root staff  13315 juil. 14 23:39 spfquery
```

Spamassassin, spamc, spamd and others related spamassassin packages are not installed using .deb but from source?

**Comment by stefaweb** [comment]
A copy of the install log.
[mailscanner-install.log.txt](https://github.com/mailwatch/mailwatch-install-script/files/2195681/mailscanner-install.log.txt)

Interesting stuff.

`E: Le paquet « unrar » n'a pas de version susceptible d'être installée` (3 times in log)
`make: *** [test_dynamic] Erreur 255` (3 times in log)

Plus around 10 times "Failed".

**Comment by Skywalker-11** [comment]
Spamassassin was auto installed by MailScanner. #38 should now also install it via repository

**Comment by Skywalker-11** [comment]
> E: Le paquet « unrar » n'a pas de version susceptible d'être installée (3 times in log)
make: *** [test_dynamic] Erreur 255 (3 times in log)

This comes from the default mailscanner install script

**Comment by ghost** [comment]
Is the mailwatch install script fetching the latest MailScanner package?

**Comment by Skywalker-11** [comment]
> Is the mailwatch install script fetching the latest MailScanner package?

It is defined by in setup.sh after my PR in version.inc. The script didn't have the current version (only 5.0.7-2).

**Comment by ghost** [comment]
@stefaweb  The errors you have are non-fatal and expected, mostly failed tests as a result of unexpected dns answers (this is external and usually caused by the dns servers responses or firewall configurations, not really an issue with the packages themselves).  Apparently there isn't an unrar package on debian?  Or is there and it is called something else?

**Comment by stefaweb** [comment]
Versions
mw 1.2.8
ms 5.0.7-2

Can be have the last one instead?

**Comment by Skywalker-11** [comment]
Package on debian is `unrar` from non-free repositories https://packages.debian.org/search?keywords=unrar&searchon=names&section=all
or `unrar-free` https://packages.debian.org/search?keywords=unrar-free&searchon=names&section=all

**Comment by ghost** [comment]
@Skywalker-11 Thanks!  I'll need to test unrar-free and see about including it...

**Comment by ghost** [comment]
@stefaweb unrar not a big deal, should be ok to install by hand from non-free repo.

**Comment by stefaweb** [comment]
> Apparently there isn't an unrar package on debian? Or is there and it is called something else?


We have this with Stretch.
`Réception de:1 http://ftp.fr.debian.org/debian stretch/non-free amd64 unrar amd64 1:5.3.2-1+deb9u1 [127 kB]`

but we must have non-free in /etc/apt/sources.list

**Comment by stefaweb** [comment]
Solved with #38.
